### ISSUE TITLE
Tests failing

### ISSUE THREAD

**Issue by susilehtola** [psi4/psi4#3028]
The following smoketests are failing on the current master branch
```
The following tests FAILED:
	138 - dfmp2-1 (Failed)
	486 - ddd-deriv (Failed)
	491 - psi4numpy-rhf (Failed)
	493 - psi4numpy-rhf-hessian (Failed)
	498 - python-energy (Failed)
	513 - json-schema-1-gradient (Failed)
```
Are these failures reproducible?

**Comment by loriab** [comment]
Have you pulled tags after the latest release? `git fetch upstream "refs/tags/*:refs/tags/*"`

**Comment by susilehtola** [comment]
Yup

**Comment by loriab** [comment]
I've got a branch that's only two days removed from master, and its smoke tests are clean. There is a problem with psi4 and qcng 0.28, but I'm sure you're on linux.

After pulling tags, one does have to rebuild for the versioning to take effect. But if you're sure that's not it, go ahead and execute the results of `<objdir>/stage/bin/psi4 --psiapi`. It shouldn't be necessary for ctest, but considering those failures are those tests running the python command _and_ those that use qcengine, best to make sure of paths. Beyond that, see if `<objdir>/Testing/Temporary/LastTest.log` has some output clues.

**Comment by susilehtola** [comment]
> I've got a branch that's only two days removed from master, and its smoke tests are clean. There is a problem with psi4 and qcng 0.28, but I'm sure you're on linux.

Yeah but that's the point, I am not sure if this is related to the `std::vector` cleanups. Just wanted for someone to check whether master is still clean

**Comment by loriab** [comment]
Ok, yes, I just ran a full tests on today's master, and it's clean. (And the Windows issue is fixed, too.)
